tools/topology/sof/pipe-processing-playback.m4

```diff
@@ -48,15 +48,6 @@ C_CONTROLBYTES(Post Process Setup Config, PIPELINE_ID,
 	,
 	PP_SETUP_CONFIG)
 
-
-# Post process runtime params
-CONTROLBYTES_PRIV(PP_RUNTIME_PARAMS,
-`	bytes "0x53,0x4f,0x46,0x00,0x01,0x00,0x00,0x00,'
-`	0x00,0x00,0x00,0x00,0x00,0x10,0x00,0x03,'
-`	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
-`	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00"'
-)
-
```

Comment:
Commit message has typo: should be "delete".

Reply:
whelp, can't even saying I was "deleting characters." thanks.

Reply:
@cujomalainey, why do you want to remove this?